common/kvstore/src/test/java/org/apache/spark/util/kvstore/DBIteratorSuite.java

```diff
@@ -383,7 +383,7 @@ public void testRefWithIntNaturalKey() throws Exception {
     LevelDBSuite.IntKeyType i = new LevelDBSuite.IntKeyType();
     i.key = 1;
     i.id = "1";
-    i.values = Arrays.asList("1");
+    i.values = Collections.singletonList("1");
```

Comment:
If you are already changing this, I would  also suggest to add static imports to shorten the code

Reply:
I don't think this sort of thing is worth changing. I know, IntelliJ suggests it. Unless the mutability is an issue, I'd leave it as a shorter idiom. I wouldn't use static imports here personally.

there's also a very small cost of changes in that they create potential merge conflicts for other changes later, so I tend to have a very low but finite minimum bar for value of code scrubbing like this.

Reply:
I agree, I don't really care about moving to `Collections`.